src/EnergyPlus/PlantChillers.hh

```diff
@@ -268,6 +394,9 @@ namespace PlantChillers {
 		int HRLoopSideNum; // heat recovery water plant loop side index
 		int HRBranchNum; // heat recovery water plant loop branch index
 		int HRCompNum; // heat recovery water plant loop component index
+		bool MyFlag;
+		bool MyEnvrnFlag;
+		ElectricReportVars reports;
```

Comment:
@Myoldmopar Why are you using a ReportVars variable here? I thought we were wanting to put them as member variables?